accounts/views.py

```diff
@@ -1,9 +1,9 @@
 from django.contrib.auth.forms import UserCreationForm
 from django.urls import reverse_lazy
-from django.views import generic
+from django.views.generic import CreateView
 
 
-class SignUpView(generic.CreateView):
+class SignUpView(CreateView):
```

Comment:
Why did you change the import?

Reply:
It is the same object
![image](https://user-images.githubusercontent.com/76876421/170834886-5743049c-2c68-484e-a05c-a50484db1d5c.png)

Reply:
Same one, Why did you change the import?